sort_requirements/script.py

```diff
@@ -70,6 +70,7 @@ def main():
         with open(path, "r") as f:
             original = f.read()
             modified = sort_requirements(original)
+            modified = "\n".join(set(modified.split("\n")))
```

Comment:
doing this likely results in the output being out of order

Reply:
Hmm. I was probably a victim of my own refactor, since I first removed the dups *before* sorting. I relied on test, which keep running OK on my system :thinking:

Reply:
yeah the problem with sets is they may or may not be in the order specified so it's unreliable for sorting.

Reply:
True. Latest commit should take care of that, as I use the set intermediately, and **before**  sorting. Local tests remained green but circle keeps failing.

Reply:
oh, it's something different now. Probably trailing blank lines. Not sure why I don't get this to fail locally, though.